### PR TITLE
[1] Cleaned Billing Code to Work With Latest Schema

### PR DESCRIPTION
**Pull request by aivantg**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

Preview: https://deploy-preview-118--peoplepower.netlify.com/

## What's new in this PR?

[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'

- Rewrote Subscriber Utils to simplify transaction logic immensely
- Rewrote TransactionsTable to be generalized across screens
- Rewrote Billing page for updated designs and updated schema
- Rewrote Subscriber Dashboard Billing Card for updated schema
- Small Cleanup Fixes (Better Date Formatting, cleaner subscriber dashboard, etc)

Strangely this PR includes commits from #115 because i didn't branch off properly. I don't think it's a big deal because squash and merge + files changed seems accurate

[//]: # '############## END ##################'

### How to Review

[//]: # 'The order in which to review files and 
what to expect when testing locally'
[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'

Logging in as lorajofoo@gmail.com, you should see the two outstanding bills and a payment!
Log in as sam@berkeley.edu, and you should see no bills but many payments (unrealistic but there lol)

[//]: # '############## END ##################'

### Related PRs

[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'

PR to add Payment flow to billing page and subscriber dashboard

[//]: # '############## END ##################'

### Tests Performed, Edge Cases

[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Screenshots

[//]: # '#####################################'
[//]: # '#### YOUR SCREENSHOTS BELOW ########'
[//]: # '############## END ##################'

CC: @dfangshuo

### CODE REVIEW

File: src/screens/shared/components/AnnouncementList.js [hunk 1]
```diff
@@ -26,7 +26,10 @@ const AnnouncementList = props => {
   });
 
   return (
-    <div className={css.concat(' ', 'cardsCont')} style={{ overflow: 'auto' }}>
+    <div
+      className={(css || '').concat(' ', 'cardsCont')}
```
Comment (dfangshuo) [inline src/screens/shared/components/AnnouncementList.js:30]: question: what does this line do?

File: src/lib/subscriberUtils.js [hunk 2]
```diff
@@ -1,107 +1,54 @@
...
+});
+
+const createTransactionFromBill = bill => ({
+  balance: formatAmount(bill.balance),
+  date: moment(bill.statementDate).format(TRANSACTION_DATE_FORMAT),
+  description: `${moment(bill.startDate).format('MMMM')} Power Bill`,
+  charge: formatAmount(bill.currentCharges),
+  payment: '',
+  amount: formatAmount(bill.currentCharges)
+});
```
Comment (dfangshuo) [inline src/lib/subscriberUtils.js:26]: this is beautiful ++

File: src/screens/shared/components/AnnouncementList.js (hunk 1 above)
Comment (aivantg) [reply src/screens/shared/components/AnnouncementList.js:30]: umm.. tbh not sure where this change came form 😬
